### DISCUSSION TITLE
[Script request] Portainer

### DISCUSSION THREAD

**Discussion by Mightyretro** [community-scripts/ProxmoxVE#479] [Request script] [answered]
### Application Name

Portainer

### Website

https://portainer.io/ & https://github.com/portainer/portainer

### Description

Portainer Community Edition is a lightweight service delivery platform for containerized applications that can be used to manage Docker, Swarm, Kubernetes and ACI environments. It is designed to be as simple to deploy as it is to use. The application allows you to manage all your orchestrator resources (containers, images, volumes, networks and more) through a ‘smart’ GUI and/or an extensive API.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Mati-l33t** [comment, marked as answer]
Docker script have options to Install Portainer and/or Docker Compose V2.

https://community-scripts.github.io/ProxmoxVE/scripts?id=docker

**Reply by Mightyretro** [reply]
Thank you ))
